texastribune/donations
# Temporary Safari 12.1 fix

#### What's this PR do?
+ Forces Vue to re-render so Safari payments work
+ Removes Safari user message

#### Why are we doing this? How does it help us?
Money.

#### How should this be manually tested?
On Safari 12.1, ensure the following things work:
+ Payments on `/donate`. Ensure red messages appear when form inputs are invalid. Try various `?installmentPeriod=<monthly><yearly><once>&amount=<amount>` combinations.
+ Same thing on `/circle`, except with these query params: `?level=<editorMonthly><editorYearly><chairmanMonthly><chairmanYearly><leadershipMonthly><leadershipYearly>`.
+ Same thing on `/business`, except no need to try various query params.
+ Try making payments on at least one other browser for sanity.

#### How should this change be communicated to end users?
Removing the message & will let audience know.

#### Are there any smells or added technical debt to note?
Lol.

#### What are the relevant tickets?
Off-sprint.

#### Have you done the following, if applicable:

#### TODOs / next steps:
Submit bug report to Vue + Stack Overflow.
